app/templates/releases/lts.hbs

```diff
@@ -83,78 +83,19 @@
       <th>LTS version</th>
       <th>Last minor release</th>
       <th>Last minor release date</th>
+      <th>Bugfixes until</th>
+      <th>Security patches until</th>
     </tr>
   </thead>
   <tbody>
-    <tr>
-      <td>4.8</td>
-      <td>4.8.6</td>
-      <td>June 2023</td>
-    </tr>
-    <tr>
-      <td>4.4</td>
-      <td>4.4.5</td>
-      <td>May 2023</td>
-    </tr>
-    <tr>
-      <td>3.28</td>
-      <td>3.28.11</td>
-      <td>November 2022</td>
-    </tr>
-    <tr>
-      <td>3.24</td>
-      <td>3.24.6</td>
-      <td>November 2021</td>
-    </tr>
-    <tr>
-      <td>3.20</td>
-      <td>3.20.3</td>
-      <td>August 2020</td>
-    </tr>
-    <tr>
-      <td>3.16</td>
-      <td>3.16.10</td>
-      <td>August 2020</td>
-    </tr>
-    <tr>
-      <td>3.12</td>
-      <td>3.12.4</td>
-      <td>May 2020</td>
-    </tr>
-    <tr>
-      <td>3.8</td>
-      <td>3.8.3</td>
-      <td>June 2019</td>
-    </tr>
-    <tr>
-      <td>3.4</td>
-      <td>3.4.8</td>
-      <td>January 2019</td>
-    </tr>
-    <tr>
-      <td>2.18</td>
-      <td>2.18.2</td>
-      <td>February 2018</td>
-    </tr>
-    <tr>
-      <td>2.16</td>
-      <td>2.16.4</td>
-      <td>February 2018</td>
-    </tr>
-    <tr>
-      <td>2.12</td>
-      <td>2.12.2</td>
-      <td>April 2017</td>
-    </tr>
-    <tr>
-      <td>2.8</td>
-      <td>2.8.3</td>
-      <td>November 2016</td>
-    </tr>
-    <tr>
-      <td>2.4</td>
-      <td>2.4.5</td>
-      <td>April 2016</td>
-    </tr>
+    {{#each this.noLongerMaintainedLTS as |release|}}
```

Comment:
Thank you so much for tackling this!

I think it may be better to keep this as a static table, since the values, once added, do not change.